test/e2e/performanceprofile/functests/utils/consts.go

```diff
@@ -24,7 +26,16 @@ var NodesSelector string
 // ProfileNotFound is true when discovery mode is enabled and no valid profile was found
 var ProfileNotFound bool
 
+// Complete file name for the latency tools executable
+var LatencyToolsPath string
+
 func init() {
+
+	LatencyToolsPath = os.Getenv("LATENCY_TOOLS_PATH")
```

Comment:
Usually https://pkg.go.dev/os#LookupEnv is better because it help differentiate between an empty value and unset value

Reply:
Yeah. I used that but as the logic in case of variable being unset or being empty is the same, use default value, I changed it to `GetEnv`

Reply:
make sense